src/Microsoft.Data.SqlClient/tests/FunctionalTests/SslOverTdsStreamTest.cs

```diff
@@ -0,0 +1,278 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.Tests
+{
+    public static partial class SslOverTdsStreamTest
+    {
+        [Theory]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
I suggest moving all tests in same class and tagging as done here.

Reply:
There can be no equivalent test on netfx because the class doesn't exist there. If the project were built against netfx the same test could be enabled just by removing this Skip annotation, and in that case only the core parts separated out would need to be compiled out, as they are currently.